tests/_system/test_onedim_lambertian_brf.py

```diff
@@ -6,6 +6,7 @@
 import numpy as np
 
 import eradiate
+from eradiate import unit_registry as ureg
```

Comment:
Do you really need to add this import here?

Reply:
No I don't, thanks ^^